docs/mccleary.md

```diff
@@ -0,0 +1,35 @@
+# nf-core/configs: McCleary Configuration
+
+All nf-core pipelines have been successfully configured for use on the McCleary at Yale University.
+
+To use, run the pipeline with `-profile mccleary`. This will download and launch the [`mccleary.config`](../conf/mccleary.config) which has been pre-configured with a setup suitable for the McCleary cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+## Below are non-mandatory information e.g. on modules to load etc
+
+Before running the pipeline you will need to load Java 11 using the environment module system on McCleary. You can do this by issuing the commands below:
+
+```bash
+## Load Java 11 environment modules
+module purge
+module load Java/17.0.4
+```
+
+## Recommendation on environment variables to set
+
+Although not required, it is recommended to set certain Nextflow specific environment variables to make better use of cluster resources.
+
+```bash
+NXF_OPTS="-Xms500M -Xmx8G"
+NXF_WORK="$PALMER_SCRATCH"
+NXF_SINGULARITY_CACHEDIR="$PALMER_SCRATCH"
+```
```

Comment:
```suggestion


There is currently a Batch file size submission limit in the McCleary cluster, if you're running into the error `sbatch: error: Batch job submission failed: Pathname of a file, directory or other parameter too long`, you can set the environment variables:

```bash
export NXF_WRAPPER_STAGE_FILE_THRESHOLD='40000'
```
This will only work for Nextflow versions >= 23.10.0 so you might need to update Nextflow first.

```